caom2repo/caom2repo/core.py

```diff
@@ -89,13 +89,17 @@
 from six.moves.urllib.parse import urlparse
 
 # from . import version as caom2repo_version
-from . import version
+from caom2repo import version
 
 __all__ = ['CAOM2RepoClient']
 
 BATCH_SIZE = int(10000)
-SERVICE_URL = 'www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/' #TODO replace with SERVICE_URI when server supports it
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f" #IVOA dateformat
+
+CAOM2REPO_FEATURE_ID = 'vos://cadc.nrc.ca~vospace/CADC/std/CAOM2Repository#obs-1.0'
```

Comment:
Why is this a 'FEATURE_ID' ?  Isn't this the CAOM2REPO_STANDARD_ID value?  When I look at the caom2repo capabilities I see there are 4 capabilities available ; capabilities, availability, LOGGING, and CAOM2Reporsitory. It's not clear from those names which one is the one you put and get records from... but FEATURE looks a bit off to me as a name.